### PR TITLE
Install LLVM, gflags, glog, tmux, and zsh.

### PR DESCRIPTION
**Pull request by xksteven**
Address issues #6, #7, and #9.

### CODE REVIEW

File: playbooks/roles/cais-bastion-head/tasks/ol-7.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Install tmux
```
Comment (AndriyNovykov) [inline playbooks/roles/cais-bastion-head/tasks/ol-7.yml:2]: The name is 'Install tmux' but we are installing SCL for GCC v7.

Comment (xksteven) [reply playbooks/roles/cais-bastion-head/tasks/ol-7.yml:2]: Thanks for the catch! Yup I messed this one up.

File: playbooks/cais_bastion_head.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+- hosts: all
```
Comment (xksteven) [inline playbooks/cais_bastion_head.yml:1]: this still needs fixing

Comment (xksteven) [reply playbooks/cais_bastion_head.yml]: fixed

**Review by AndriyNovykov — APPROVED** [review]
LGMT!

**Comment by xksteven** [comment]
Please expand on your acronym at some point lol

**Pull request merged**